src/shared/dlt_offline_trace.c

```diff
@@ -64,6 +64,8 @@
 #include <dirent.h>
 
 #include <dlt_offline_trace.h>
+#include <syslog.h>
```

Comment:
Could you please reorder the includes, first general includes, then dlt related ones?